projects/graph/src/graph_demo.py

```diff
@@ -5,12 +5,84 @@
 """
 
 from sys import argv
+from graph import Graph
+from draw import BokehGraph
+import random
 
+def createDefaultGraph():
+    graph = Graph()  
+    graph.add_vertex('0')
+    graph.add_vertex('1')
+    graph.add_vertex('2')
+    graph.add_vertex('3')
+    graph.add_vertex('4')    
+    graph.add_vertex('5')
+    graph.add_vertex('6')
+    
+    graph.add_edge('0', '1')
+    graph.add_edge('0', '3')
+    graph.add_edge('1', '2')
+    graph.add_edge('2', '4')
+    graph.add_edge('4', '5')
+    graph.add_edge('5', '6')
+        
 
-def main():
-    pass  # TODO
+    graph.dbft(-1)
+    bg = BokehGraph(graph)
+    bg.draw()
+
+def createRandomGraph(numNodes):
+    graph = Graph()
+
+    edges = []
+    
+    for i in range(numNodes):
```

Comment:
👍 this is nice. I see you're passing in edges to your main function, but i don't see it being used anywhere.